src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -291,6 +291,16 @@ public void execute()
          }
       });
       
+      widget_.addHandler(new PasteEvent.Handler()
```

Comment:
Why doesn't this attach it's code to the existing addPasteHandler registration on line 275?